packer/core.go

```diff
@@ -397,6 +397,8 @@ func (c *Core) init() error {
 		allVariables[k] = v
 	}
 
+	// One idea: Don't interpolate variables
+
```

Comment:
Here all the variables content from the template and from the cli are interpolated. My question is, do we want/need to interpolate these before everything? If we don't do this here, we gonna need double interpolation later.